compiler/src/printFexpr.mli

```diff
@@ -1,3 +1,3 @@
-val pp_fexpr : Format.formatter -> Fexpr.fexpr -> unit
-val pp_rexpr : Format.formatter -> Fexpr.rexpr -> unit
-val pp_lexpr : Format.formatter -> Fexpr.lexpr -> unit
+val pp_fexpr : debug:bool -> Format.formatter -> Fexpr.fexpr -> unit
+val pp_rexpr : debug:bool -> Format.formatter -> Fexpr.rexpr -> unit
+val pp_lexpr : debug:bool -> Format.formatter -> Fexpr.lexpr -> unit
```

Comment:
Does this new argument have any effect? When reaching `linear`, are there any casts to `int` left?

Reply:
No effect on the tests at least, so I removed the argument. Thanks for the remark!